userspace/libsinsp/thread_group_info.h

```diff
@@ -57,7 +57,7 @@ typedef struct thread_group_info
 		 * real expired threads, they are just the ones marked as dead. For example the main thread
 		 * of the group is marked as dead but it will be never expired until the thread group exists.
 		 */
-		if((m_threads.size() - m_alive_count) > DEFAULT_DEAD_THREADS_THRESHOLD)
+		if((m_threads.size() - m_alive_count) >= DEFAULT_DEAD_THREADS_THRESHOLD)
```

Comment:
Same: why is this a fix?

Reply:
Same reason for this https://github.com/falcosecurity/libs/pull/1182#discussion_r1267927855